src/ipsis/course-merge/CourseMergeMergedOfferingCollectionEntity.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * SelectedCourseMergeOfferingCollectionEntity class representation of a list of course merge offering as defined in the LMS
+ * See: ISirenCourseMergeSerializer.SerializeSelectedCourseOfferingListResult
+ */
+import { Actions, Classes } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class CourseMergeMergedOfferingCollectionEntity extends Entity {
+	originalSourceCourseMergeOfferings() {
+		if (!this._entity) {
```

Comment:
I think we could just do this instead of having this check? Similar to the other functions.
`return this._entity?.entities?.filter(course => !course.class.includes(Classes.ipsis.originalTarget));`